index.js

```diff
@@ -74,20 +72,34 @@ function onModelChange(e) {
     }));
 }
 
-var RovingTabindex = function RovingTabindex(el) {
-    _classCallCheck(this, RovingTabindex);
+var RovingTabindex = function () {
+    function RovingTabindex(el) {
```

Comment:
I guess this was due to the Babel update. Just making a note of it incase we run into any issues with it (but I doubt it)